src/main/include/RobotMap.h

```diff
@@ -45,5 +45,16 @@ constexpr int kIDCargoPneumaticReverse {8};
 //Climber Pnuematics
 constexpr int kIDClimberForward {1};
 constexpr int kIDClimberReverse {5}; 
+
+static const int kArmPot = 1;
```

Comment:
`static const int kArmPot = 1;` Should be `kIDArmPot`

`constexpr kIDArmPot {1}`;